modules/publication/README.md

```diff
@@ -0,0 +1,45 @@
+# Publication
+
+## Purpose
+The publication module allows users to propose research projects based on LORIS 
+data, upload publication media, and allows lead study PIs to review the project 
+proposals.
+
+## Intended Users
+
+Publication module is intended for site PIs to propose specific research projects
+and for lead PIs to review them. When proposing a project, PIs may also specify 
+other LORIS users to help with editing and maintaining their project proposals.
+
+## Scope
+
+The publication module is intended to view and manage research project proposals.
+Users who are proposing a project are able to specify which specific variables of 
+interest they are interested in studying. However, gathering and releasing this 
+data remains within the scope of the DQT and Data Release modules respectively.
+
+## Permissions
+
+There are three main permissions for the publication module:
+
+- `publication_view`: grants the user access to viewing the module and pending 
+or approved project proposals, as well as downloading publication media which may 
+be included in the proposals
+- `publication_propose`: grants the user access to proposing their research projects
+and uploading relevant publication media
+- `publication_approve`: grants the user access to reviewing (accepting or rejecting)
+project proposals. Only users with this permission are allowed to view rejected project
+proposals.
+
+## Configurations
+
+- `publication_uploads`: This configuration determines the directory where file uploads
+ get stored. By default, this directory will be `/data/publication_uploads/` but is not 
+ created 
+
+## Interactions with LORIS
+
+The publication module creates foreign key relations between its primary `publication` 
+ table, the `parameter_type` table, the `test_names` table, and the `users` table.
+ There is also a link to the Data Dictionary in the project proposal / editing view
+ in order to assist in finding variables of interest.
```

Comment:
Are there any links to the DQT or Data Release module as implied in the scope?

Reply:
> Are there any links to the DQT or Data Release module as implied in the scope?

couldn't think of any good place to put them / define how that workflow should play out :p